invisible_cities/reco/calib_sensors_functions_test.py

```diff
@@ -247,6 +247,7 @@ def test_wf_baseline_subtracted_is_close_to_zero(gaussian_sipm_signal):
 def test_mean_for_pmts_fee_is_unbiased(square_pmt_and_sipm_waveforms):
     _, _, pmts_fee, _, _, _, _ = square_pmt_and_sipm_waveforms
     pmts_bls = csf.subtract_mean(pmts_fee) # ped subtracted near zero
+    print(pmts_bls)
```

Comment:
I assume that this `print` is a hangover that will be eliminated before merging.

Reply:
Yep, sorry, was trying to understand the fail I was getting and forgot to remove it